TwitchChannelPointsMiner/classes/WebSocketsPool.py

```diff
@@ -215,11 +215,13 @@ def on_message(ws, message):
                         elif message.type == "stream-down":
                             if ws.streamers[streamer_index].is_online is True:
                                 ws.streamers[streamer_index].set_offline()
-                        elif message.type == "viewcount":
-                            if ws.streamers[streamer_index].stream_up_elapsed():
-                                ws.twitch.check_streamer_online(
-                                    ws.streamers[streamer_index]
-                                )
+                        elif (
```

Comment:
Here for example not sure the change is really useful. if `ws.streamers[streamer_index].stream_up_elapsed()` is false, then all the other `elif`s will be checked and are sure to fail as message type should match here. Before, it wasn't checking them all.

The semantic is lost. We first wanna grab the message type, and apply some kind of treatment to it.

IMO to avoid nested ifs there, there should be some kind of "handler" that handles a ws message. Each one can be a class or a method. For example there could be "onViewCount", "onRaidUppdate"... That would allow to split code in more understandable sections instead of a big if switch.